src/index.js

```diff
@@ -432,6 +432,36 @@ class MiniCssExtractPlugin {
           return source;
         }
       );
+
+      compilation.hooks.optimizeChunks.tap(pluginName, (chunks) => {
+        for (const chunk of chunks) {
+          chunk.modulesIterable.forEach((module) => {
```

Comment:
An iterable has no forEach method. You should not assume that it's an Set.

Use `for of` instead.